packages/base/src/sizedMixin.ts

```diff
@@ -73,11 +73,11 @@ export function SizedMixin<T extends Constructor<ReactiveElement>>(
 
         private _size: ElementSize | null = defaultSize;
 
-        protected firstUpdated(changes: PropertyValues): void {
-            super.firstUpdated(changes);
+        protected update(changes: PropertyValues): void {
```

Comment:
Would `willUpdate` be a better fit here? It'd remove the need for `super` and sounds from the [docs](https://lit.dev/docs/components/lifecycle/#willUpdate) like it may exist for essentially this kind of use case.